lib/compare.rb

```diff
@@ -390,7 +390,7 @@ def complete_runs(cms = nil)
       cms.map { |m| matrix_cols m }
     end
 
-    def get_all_stat_keys
+    def retrieve_all_stat_keys
```

Comment:
thanks, help rename this by removing get_ prefix, so it can be def all_stat_keys. Similar comments to other get_ methods.